go/vt/vtgate/router.go

```diff
@@ -281,7 +285,8 @@ func (rtr *Router) execUpdateEqual(vcursor *requestContext, route *engine.Route)
 	if len(ksid) == 0 {
 		return &sqltypes.Result{}, nil
 	}
-	rewritten := sqlannotation.AddKeyspaceID(route.Query, ksid, vcursor.comments)
+	keyspaceIds = append(keyspaceIds, ksid)
+	rewritten := sqlannotation.AddKeyspaceIDs(route.Query, keyspaceIds, vcursor.comments)
```

Comment:
This can be simplified to: `rewritten := sqlannotation.AddKeyspaceIDs(route.Query, [][]byte{ksid}, vcursor.comments)`. Same comments below.